drivers/can/Kconfig.mcp2515

```diff
@@ -20,34 +20,6 @@ config CAN_MCP2515_OSC_FREQ
 	help
 	  Specify the frequency of the oscillator connected to the MCP2515.
 
-config CAN_PROP_SEG
-	int "Prop_Seg"
-	default 2
-	range 1 8
-	help
-	  Time quanta of propagation segment (ISO 11898-1)
```

Comment:
I think it is not correct to move this configs to device tree, does it depend on hardware design or is it software option which can be changed at given hardware?

Reply:
These are properties of the hardware. More specific, the transceiver, wiring, and other delays on the board.  It should have been in the device tree from the beginning.

Reply:
Welllll now they are 4 properties ;-) But yes, much nicer this way.
Also both KConfig and devicetree hardcode it, so it's not changing much in behavior